src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
```

Comment:
I would set pressedKey state to null, and update it so it's only the key name and I would write condition in rendered JSX that returns the right string depending on pressedKey.

Reply:
I would leave it as is, both solutions are fine